runtimes/huggingface/mlserver_huggingface/codecs/base.py

```diff
@@ -181,6 +181,11 @@ def decode_request(cls, request: InferenceRequest) -> Dict[str, Any]:
 
             value = get_decoded_or_raw(item)
             values[item.name] = value
+        if request.parameters is not None:
```

Comment:
I suggest to have a unit test for this extra logic on the (hf) codec level

Reply:
added a unit test for this in `runtumes.huggingface.tests.test_codecs.py::test_encode_request`

Reply:
I think you meant `test_decode_request`